dtdiff

```diff
@@ -17,7 +17,7 @@ source_and_sort () {
 	    *.dts)
 		IFORMAT=dts
 		;;
-	    *.dtb)
+	    *.dtb|*.dtbo)
```

Comment:
Even something this simple needs some sort of commit message: why is this change desirable?  (*I* can see why, but someone reading the commit log in the future might not).